### DISCUSSION TITLE
errors when creating code

### DISCUSSION THREAD

**Discussion by JobeMondisa** [community/community#57072] [Programming Help]
i would like to know if i need to install a software or do some settings first before i do coding on cloud vs code.
im currently on the week1 and i get an error when i type (make hello).

**Comment by Bibiwei-Pere** [comment]
Hi @JobeMondisa 

Which language are you using for your site?

If it's HTML, CSS and JavaScript then basically you'll need to install some extensions for your code to start working.

Open your vs code and click extension from the sidebar on the left.

Search and Install the following extensions.
1. Live preview
     Allows you view your code in browser
2. Prettier
    Helps Format your code. After installing, head 
    over to settings and search for formatter then 
    select Prettier from the options. Then click edit 
    on save

There are many other extension but these are the basic ones for html, css and JavaScript.

For more info check out https://www.freecodecamp.org/news/how-to-set-up-vs-code-for-web-development/amp/

**Reply by JobeMondisa** [reply]
i use their codespace via Gitrhub, was told everything is pre installed.
im not sure about the language, i believe is C++.

**Reply by Bibiwei-Pere** [reply]
@JobeMondisa 

Firstly, it's very important for you to know the language you are coding with because an html code will not work if written in a css file, likewise python code will not work if written in a JavaScript file. 

Secondly, are you trying to preview the code on your terminal or web browser.

Lastly, is it possible for you to send your code or a link to it here so we can easily pin point the source of the error.

Thanks.

**Comment by JobeMondisa** [comment]
Hi Bibiwie,
thank you so much for your help.

I have just found out that the language I'm using is syntax and I'm using a
web browser.

I have not found a solution yet.
Regards

Jobe

**Reply by davea42** [reply]
Normally the word 'syntax' means following the rules defined for the language in use,
and by rules I mean what one sees looking at code.   It is a generic term on its own
and not really have meaning unless one knows which specific language
syntax applies.

Typical (computer) languages are  C C++ C+ Java css javascript python ... and
many others.  Every one of these languages has its own particular
syntax.

We have not seen anything yet (a sample?) to indicate what
the language is (and we will then know what the syntax rules are).
Hope this helps.
